c2corg_ui/static/js/maincontroller.js

```diff
@@ -78,7 +78,7 @@ app.MainController.prototype.isPath = function(path) {
     return 'home';
   // if topoguide, it can be all kinds of documents
   } else if (path === 'topoguide') {
-    return location.indexOf('waypoints') > -1 || location.indexOf('routes') > -1 || location.indexOf('outings') > -1;
+    return location.indexOf('waypoints') > -1 || location.indexOf('routes') > -1 || location.indexOf('outings') > -1 || location.indexOf('articles') > -1;
```

Comment:
OK thanks.

So it means:
* ``areas`` should be added to the list for ``topoguide`` => @dkocich could you do it in this PR?
* ``articles`` should not be added to the list for ``topoguide`` since they have their own menu entry => should another path condition be added?
Something like
```
} else if (path === 'articles') {
```
?